backend/src/apiserver/common/const.go

```diff
@@ -55,6 +55,8 @@ const DefaultTokenReviewAudience string = "pipelines.kubeflow.org"
 
 const (
 	DefaultPipelineRunnerServiceAccount = "pipeline-runner"
+	DefaultMLPipelineAddress            = "ml-pipeline.kubeflow"
+	DefaultMLPipelineGRPCPort           = "8887"
```

Comment:
nitpick: should these have `Service` in the name to match the above (`mlPipelineServiceAddress`, `mlPipelineServiceGRPCPort`)? Couldn't tell if maybe these weren't always services / the name was intentional.